openedx/core/djangoapps/catalog/management/commands/sync_course_runs.py

```diff
@@ -0,0 +1,55 @@
+"""
+Sync course runs from catalog service.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from opaque_keys.edx.keys import CourseKey
+
+from catalog.utils import get_catalog_course_runs
+from content.course_overviews.models import CourseOverview
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_catalog_course_runs():
+    """
+    Fetch catalog course runs from catalog service for all course overview records.
+    """
+    course_overviews = CourseOverview.objects.all()
```

Comment:
Hmm.. rather than getting all the courses from CourseOverview and querying CourseCatalog with it, I suggest getting all the courses from CourseCatalog and updating CourseOverview accordingly.

Reasons:

1. The number of courses in `CourseCatalog` < number of courses in `CourseOverview` since the former has only market-facing courses and doesn't include any test courses. 
2. Also, CCX courses are in `CourseOverview` but not in `CourseCatalog`.  For example, Edge has 839 CCX courses, while Prod has 43, per query: `select count(id) from  course_overviews_courseoverview where _location like "%ccx%";`
3. Eventually, when you handle CCX, you'll want to check whether any CCX courses exist for every course from the Catalog and update its CourseOverview entry accordingly.